resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesDriverBuilder.scala

```diff
@@ -81,14 +81,14 @@ private[spark] class KubernetesDriverBuilder(
 
     val driverCommandStep = provideDriverCommandStep(kubernetesConf)
 
-    val maybeHadoopConfigStep =
-      kubernetesConf.hadoopConfSpec.map { _ =>
-        provideHadoopGlobalStep(kubernetesConf)}
+    val otherSteps = Seq(
```

Comment:
nit: `otherStep` isn't very descriptive to the contents of the steps. I prefer `hadoopSteps` if anything

Reply:
All this code will go away when I get to SPARK-25877, so I'm not super concerned with the names here.